cmd/apply.go

```diff
@@ -26,12 +26,14 @@ import (
 	"io/ioutil"
 
 	"github.com/spf13/cobra"
+	sppb "google.golang.org/genproto/googleapis/spanner/v1"
```

Comment:
This `cmd` package should be agnostic for the actual `spanner` API spec (proto). Instead, we should delegate such specific knowledge to the `pkg/spanner` package.

So, please make `pkg/spanner.ApplyPartitionedDML` take the priority as int (iota) and also make `pkg/spanner` package provide some constatant like `PriorityTypeXXX`.

Reply:
done 874770e